### PR TITLE
TUA-980 Verify that user can add center for the existing club

### LINKED ISSUE #322: TUA-980 Verify that user can add center for the existing club

### PR DESCRIPTION
**Pull request by Natalia62**

### CODE REVIEW

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java [hunk 1]
```diff
@@ -70,12 +74,18 @@ public class AddClubPopUpStepOne extends AddClubPopUpContainer {
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-item']")
     private WebElement centerSelectedTitle;
 
-    @FindBy(xpath = "//div[@class='ant-select-item-option-content']")
+    @FindBy(xpath = "//div[@class='ant-select-item ant-select-item-option']")
```
Comment (OksankaV) [inline src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: This xpath miss selected item of the list. It is better use 'contains'
//div[contains(@class,'ant-select-item ant-select-item-option')]

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java [hunk 2]
```diff
@@ -70,12 +74,18 @@ public class AddClubPopUpStepOne extends AddClubPopUpContainer {
...
     private List<WebElement> centersList;
 
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-placeholder']")
     private WebElement selectPlaceholder;
 
+    @FindBy(xpath = "./descendant::div[contains(@class, ' add-club-select')]")
+    private WebElement centerSelectedInEditClub;
```
Comment (OksankaV) [inline src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: Duplicate xpath, we have centerSelect
It is better to change xpath for centerSelect 
"./descendant::input[contains(@id,'centerId')]"

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java [hunk 3]
```diff
@@ -112,4 +122,19 @@ public AddClubPopUpStepOne clickCenterDropdown(){
         return this;
     }
 
+    @Step("Click on the dropdown with a list of centers")
+    public AddClubPopUpStepOne clickEditCenterDropdown(){
+        centerSelectedInEditClub.click();
+        return this;
+    }
```
Comment (OksankaV) [inline src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: duplicate method, we have clickCenterDropdown for click

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java [hunk 4]
```diff
@@ -112,4 +122,19 @@ public AddClubPopUpStepOne clickCenterDropdown(){
...
+
+    @Step("Select new center for club")
+    public AddClubPopUpStepOne changeCenterForClub(String value) {
```
Comment (OksankaV) [inline src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: duplicate method, we have selectCenter for select center

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java (hunk 3 above)
Comment (Natalia62) [reply src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: Removed the duplicate, updated the clickCenterDropdown method

File: src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java (hunk 4 above)
Comment (Natalia62) [reply src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java]: Removed the duplicate, updated the selectCenter method

**Comment by lhalam** [comment]
Conflicting files
src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java
src/main/java/com/academy/ui/pages/ClubPage.java
src/test/java/com/academy/ui/profilePage/EditClubCardWithManagerTest.java

**Pull request merged**